packages/kbn-telemetry-tools/BUILD.bazel

```diff
@@ -47,6 +47,7 @@ TYPES_DEPS = [
   "@npm//@types/node",
   "@npm//@types/normalize-path",
   "@npm//@types/testing-library__jest-dom",
+  "@npm//resize-observer-polyfill"
```

Comment:
This is a server-side package. It shouldn't depend on `resize-observer-polyfill`.